pwa/src/components/DiagnosisForm.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { ReactElement } from 'react'
+import { List, Checkbox, InputItem, WhiteSpace } from 'antd-mobile'
+import '../scss/DiagnosisForm.scss'
+import * as data from './DiagnosisFields.json'
+
+const CheckboxItem = Checkbox.CheckboxItem
```

Comment:
Variable declarations together after the comment.

Reply:
Resolved in [this commit](https://github.com/Haiti-Distributed-Healthcare-System/hdhs/pull/84/commits/3197b229929bb40c992e71084d6e3cd667482fba).